src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,10 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        PreferencesManager.on("change", "uppercaseColors", function () {
+            this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        }.bind(this));
```

Comment:
Make sure new method is part of object prototype: `ColorEditor.prototype.destroy()`